README.md

```diff
@@ -11,8 +11,8 @@ Beside the Git repository, the UI framework is also available through the follow
 
 The UI framework and default skin bundled with the latest player release are always available via CDN. This is the recommended way if you just want to work with the predefined UI components. All components will be available in the `bitmovin.playerui` namespace.
 
- * JavaScript library: `//bitmovin-a.akamaihd.net/bitmovin-player/stable/7/bitmovinplayer-ui.js` 
- * CSS default skin: `//bitmovin-a.akamaihd.net/bitmovin-player/stable/7/bitmovinplayer-ui.css`
+ * JavaScript library: `//cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.js` 
+ * CSS default skin: `//cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.css`
 
 ### NPM
 
```

Comment:
I can't comment line 45:
update to 8.0 release

Reply:
It was introduced in 7.0 so this is fine. The question is rather if this description still makes sense at all, since the framework isn't reall new anymore.